VisualStudio.gitignore

```diff
@@ -263,3 +263,6 @@ paket-files/
 # Python Tools for Visual Studio (PTVS)
 __pycache__/
 *.pyc
+
+# Cake
+tools/
```

Comment:
I agree it is a possibility. There are a few of these folders in there already e.g. artifacts/ publish/. Cake is now part of the .NET Foundation and growing in popularity. This PR can go either way.